mutex/src/workflows.ts

```diff
@@ -0,0 +1,66 @@
+import * as workflow from '@temporalio/workflow';
```

Comment:
curious why namespace import vs named import. personally, i don't like having to remember while reading the code that `workflow` is a package—my brain assumes it's an object that's being operated on. and it feels like a lot of `workflow.`s to read/skip over. to me, those things are more important than the reader having to know/remember that all these functions are imported vs defined in this file.

Reply:
Copy/pasted and didn't change it. I'll switch to named imports.